CHANGELOG.md

```diff
@@ -6,15 +6,22 @@ The following is a curated list of changes in the Enact agate module, newest cha
 
 ### Added
 
+- `agate/ArcPicker` and `agate/ArcSlider` prop `disabled`
+- `agate/ArcSlider` prop `aria-valuetext`
```

Comment:
What's the purpose of this new added prop? 
for example, `agate/ArcPicker` and `agate/ArcSlider` prop `disabled` to support blah blah...

Reply:
Updated CHANGELOG.md.